front/internals/webpack/webpack.config.js

```diff
@@ -150,58 +165,64 @@ const config = {
     }),
 
     new ForkTsCheckerWebpackPlugin({
-      checkSyntacticErrors: true,
-      tsconfig: path.join(process.cwd(), 'app/tsconfig.json'),
-      silent: !!argv.json, // silent when trying to profile the chunks sizes
+      async: isDev,
+      typescript: {
+        configFile: path.join(process.cwd(), 'app/tsconfig.json'),
+      },
+      logger: { infrastructure: !!argv.json ? 'silent' : 'console' }, // silent when trying to profile the chunks sizes
     }),
 
     new CleanWebpackPlugin(),
 
     new HtmlWebpackPlugin({
       template: 'app/index.html',
       templateParameters: {
-        GOOGLE_MAPS_API_KEY: process.env.GOOGLE_MAPS_API_KEY
-      }
+        GOOGLE_MAPS_API_KEY: process.env.GOOGLE_MAPS_API_KEY,
+      },
     }),
 
     // new BundleAnalyzerPlugin(),
 
     // new webpack.ProgressPlugin(),
 
-
-
     // remove all moment locales except 'en' and the ones defined in appLocalesMomentPairs
-    !isDev && new MomentLocalesPlugin({
-      localesToKeep: [...new Set(Object.values(appLocalesMomentPairs))]
-    }),
-
-    !isDev && new MomentTimezoneDataPlugin({
-      startYear: 2014,
-      endYear: currentYear + 8,
-    }),
-
-    !isDev && new MiniCssExtractPlugin({
-      filename: '[name].[contenthash].css',
-      chunkFilename: '[name].[contenthash].chunk.css'
-    }),
-
-    !isDev && new webpack.HashedModuleIdsPlugin(),
-
-     process.env.CI && buildSourceMap && new SentryCliPlugin({
-      include: path.join(process.cwd(), 'build'),
-      release: process.env.CIRCLE_BUILD_NUM,
-    })
+    !isDev &&
+      new MomentLocalesPlugin({
+        localesToKeep: [...new Set(Object.values(appLocalesMomentPairs))],
+      }),
+
+    !isDev &&
+      new MomentTimezoneDataPlugin({
+        startYear: 2014,
+        endYear: currentYear + 8,
+      }),
+
+    !isDev &&
+      new MiniCssExtractPlugin({
+        filename: '[name].[contenthash].css',
+        chunkFilename: '[name].[contenthash].chunk.css',
+      }),
+
+    process.env.CI &&
+      buildSourceMap &&
+      new SentryCliPlugin({
+        include: path.join(process.cwd(), 'build'),
+        release: process.env.CIRCLE_BUILD_NUM,
+      }),
   ].filter(Boolean),
 
   resolve: {
     modules: [path.join(process.cwd(), 'app'), 'node_modules'],
     extensions: ['.wasm', '.mjs', '.js', '.jsx', '.ts', '.tsx', '.json'],
     alias: {
-      'polished': path.resolve('./node_modules/polished'),
-      'moment': path.resolve('./node_modules/moment'),
-      'react': path.resolve('./node_modules/react'),
+      polished: path.resolve('./node_modules/polished'),
+      moment: path.resolve('./node_modules/moment'),
+      react: path.resolve('./node_modules/react'),
       'styled-components': path.resolve('./node_modules/styled-components'),
-    }
+    },
+    fallback: {
+      util: require.resolve('util/'),
```

Comment:
We need this installed as webpack 5 no longer includes node polyfills:
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#:~:text=Webpack%205%20stops%20automatically%20polyfilling,core%20modules%20are%20not%20available.

Reply:
The short explanation they give it they removed node polyfills for the browser, do we rely on many ? Do we need to ?

Reply:
As far as I can tell, `utils` is the only one that we need so I added it. The bundling doesn't work without it. Depending on the dependencies that we add and what they in turn depend on, we might need others in the future but I doubt it.